src/libusermode/uh-private.hpp

```diff
@@ -147,17 +166,22 @@ class userhook : public pluginex
 {
 public:
     int initialized;
+    size_t* offsets;
```

Comment:
```suggestion
    std::array<size_t, __OFFSET_MAX> offsets;
```
arrays degrading to pointers are a big no no.

Reply:
+1